web.py

```diff
@@ -4,17 +4,34 @@
 from github import Github
 from joiner_scripts.joiner import AqGpsJoiner
 import datetime
+import base64
 
 app = Flask(__name__)
 
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+#Variables
+g = Github("fa0dfb5e6103c9bf91cbbcc22e3c1aa5708cbdb4")
```

Comment:
Should GitHub tokens be secret?

Reply:
Yes, they probably should be, I have been looking into storing it in a pem key. However, there does appear to be an error where every few months the key stops working, though this appears to be because of the way the key is being used. I will investigate this further.